### PR TITLE
Tweaks for MVP footer

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Tweaks for MVP footer.

## Significant changes and points to review

- changed to using the Sass variables which reference CSS variables, instead of the Sass variables which reference Sass variables (I know this is confusing, I'm confused and I wrote it)
- increased specificity on some link state style declarations
- focus style tweaks
- added spacing between list items
- moved pixelated border into footer include
- removed underline from donate button (brought up during community feedback)
- added newsletter sign up form to MVP homepage
- add dividing line between newsletter signup and footer links

## Issue / Bugzilla link

#14830

### CODE REVIEW

File: media/css/m24/components/footer-refresh.scss [hunk 1]
```diff
@@ -2,17 +2,34 @@
...
+
+@mixin divider-line {
+        display: block;
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 4px;
+        left: 0;
+        top: -10px;
+        background: $m24-color-light-gray;
```
Comment (alexgibson) [inline media/css/m24/components/footer-refresh.scss:15]: **nit:** fix indentation

File: media/css/m24/components/footer-refresh.scss [hunk 2]
```diff
@@ -173,8 +196,8 @@ $max-footer-content-width: 1440px;
 
             select {
```
Comment (alexgibson) [inline media/css/m24/components/footer-refresh.scss:197]: **Issue/suggestion:** It's impossible to distinguish that these fields are select boxes, as they appear identical to a text input? Can we add some form is chevron perhaps to help identify their function?

image

Comment (alexgibson) [reply media/css/m24/components/footer-refresh.scss:197]: **issue:** both the email field and the select inputs could also benefit from some more easily identifiable focus states?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Looks good, just a few small suggestions.

I also noticed this line of copy still need to change. It talks about news on Mozilla products which is incorrect (the signup is for Firefox news), and we should also remove the pizza reference.

image

File: media/css/m24/components/footer-refresh.scss (hunk 2 above)
Comment (stephaniehobson) [reply media/css/m24/components/footer-refresh.scss:197]: Tracking the accessibility issues in https://github.com/mozilla/bedrock/issues/14830#issuecomment-2345240527 for now, though they might need to be broken out into separate issues.

**Comment by stephaniehobson** [comment]
@alexgibson We don't have updated copy for the newsletter yet. Would it make you feel more comfortable to merge lorem ipsum than something that does not clearly need fixing?

File: media/css/m24/components/footer-refresh.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/m24/components/footer-refresh.scss:15]: ✅

**Comment by alexgibson** [comment]
@stephaniehobson happy to update the newsletter copy in a separate PR. I don't think we should do lorem ipsum as that would end up in Pontoon.

**Review by alexgibson — APPROVED** [review]

**Pull request merged**